src/sidebar/util/test/time-test.js

```diff
@@ -109,7 +109,7 @@ describe('sidebar.util.time', function() {
       const d = new Date().toISOString();
       sandbox.clock.tick(year * 2 * 1000);
 
-      assert.equal(time.toFuzzyString(d, null), 'Thu Jan 01 1970');
+      assert.equal(time.toFuzzyString(d, undefined, null), 'Thu Jan 01 1970');
```

Comment:
Seems like we should be taking advantage of being able to pass the date to compare against as an argument in these tests now. It would make these tests a lot more explicit and easier to understand.